app/src/main/java/com/example/kotlin_java_practicalss/KotlinOperatorsDemo.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.kotlin_java_practicalss
+
+fun main()
+{
+    var i =15;
```

Comment:
var i = 15,
try to provide meaningful name and don't use simcolor in kotlin code